tests/test_sinogram.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+import pytest
+from skimage import data
+from skimage.transform import radon
+
+from aspire.image import Image
+from aspire.utils import grid_2d
+
+# Relative tolerance comparing line projections to scikit
+# The same tolerance will be used in all scikit comparisons
+SK_TOL = 0.005
+
+IMG_SIZES = [
+    511,
+    512,
+]
+
+DTYPES = [
+    np.float32,
+    np.float64,
+]
+
+ANGLES = [
+    1,
+    50,
+    pytest.param(90, marks=pytest.mark.expensive),
+    pytest.param(117, marks=pytest.mark.expensive),
+    pytest.param(180, marks=pytest.mark.expensive),
+    pytest.param(360, marks=pytest.mark.expensive),
+]
+
+
+@pytest.fixture(params=DTYPES, ids=lambda x: f"dtype={x}", scope="module")
+def dtype(request):
+    """
+    Dtypes for image.
+    """
+    return request.param
+
+
+@pytest.fixture(params=IMG_SIZES, ids=lambda x: f"px={x}", scope="module")
+def img_size(request):
+    """
+    Image size.
+    """
+    return request.param
+
+
+@pytest.fixture(params=ANGLES, ids=lambda x: f"angles={x}", scope="module")
```

Comment:
I might say `f"n_angles={x}"` so these values are not confused with degrees. I would also remove the mention of degrees in the description and just say "Number of angles in radon transform." or something like that.

Reply:
Good catch Josh and I added the change. Resolving for now.